src/reducer.js

```diff
@@ -0,0 +1,109 @@
+
+import { Rectangle, RoundedRectangle, Circle } from 'components/Shapes';
+
+const SELECT_SHAPE = 'SELECT_SHAPE';
+const LOAD_SHADOW_PROPERTY = 'LOAD_SHADOW_PROPERTY';
+const UPDATE_SHADOW_PROPERTY = 'UPDATE_SHADOW_PROPERTY';
+
+const initialState = {
+  ActiveShape: Rectangle,
+  shapes: [
+    {
+      id: 'rectangle',
+      Shape: Rectangle,
+      isSelected: true
+    },
+    {
+      id: 'rounded-rectangle',
+      Shape: RoundedRectangle,
+      isSelected: false
+    },
+    {
+      id: 'circle',
+      Shape: Circle,
+      isSelected: false
+    }
+  ],
+  shadowControls: [
+    {
+      id: Math.floor(Math.random() * 100) + 1,
+      xOffset: 0,
+      yOffset: 4,
+      spread: 0,
+      blurRadius: 2,
+      opacity: 0.25,
+      shadowColor: '#000000',
+      isActive: true
+    }
+  ],
+  activeShadow: {},
```

Comment:
Similar to the comment on `ActiveShape` and `shapes`, do we need both `shadowControls` and `activeShadow`?